src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -86,16 +85,20 @@ export class NetworkRequest {
 
   constructor(
     requestId: Network.Request,
-    eventManager: EventManager,
+    networkStorage: NetworkStorage,
     cdpTarget: CdpTarget,
     redirectCount = 0
   ) {
-    this.requestId = requestId;
-    this.#eventManager = eventManager;
+    this.#requestId = requestId;
+    this.#networkStorage = networkStorage;
```

Comment:
Please inject both `networkStorage` and `eventManager` separately, I look at it and we should not have it on the networkStorage at all.

Reply:
Done